app/services/rummager_link_sync.rb

```diff
@@ -0,0 +1,27 @@
+class RummagerLinkSync
```

Comment:
No real reason, I wrote it with instantiation initially and then added the class methods as helpers.

How would you prefer it?